StackTraceParser.csproj

```
@@ -1,64 +1,43 @@
-    <RootNamespace>StackTraceParser</RootNamespace>
-    <AssemblyName>Elmah</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFramework>net452</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <NoWarn>$(NoWarn);NU5128;NU5105</NoWarn>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <LangVersion>5</LangVersion>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="nunit.framework, Version=2.6.4.14350, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
-      <HintPath>packages\NUnit.2.6.4\lib\nunit.framework.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Public.cs" />
-    <Compile Include="StackTraceParser.cs" />
-    <Compile Include="StackTraceParserTests.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="packages.config" />
-  </ItemGroup>
+
   <ItemGroup>
-    <Folder Include="Properties\" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.0" PrivateAssets="All" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" PrivateAssets="All" />
+    <PackageReference Include="NUnit" Version="3.12.0" PrivateAssets="All" />
   </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+    
   <PropertyGroup>
-    <PostBuildEvent>
-    </PostBuildEvent>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageId>StackTraceParser.Source</PackageId>
+    <Title>StackTraceParser (C# Source)</Title>
+    <PackageTags>stack trace parser diagnostics exception</PackageTags>
+    <Description>Parser for .NET and Mono stack traces.</Description>
+    <Copyright>Copyright &#169; 2020 Atif Aziz. All rights reserved.</Copyright>
+    <PackageVersion>1.3.1</PackageVersion>
+    <Authors>Atif Aziz</Authors>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <DefaultLanguage>en-us</DefaultLanguage>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PackageOutputPath>.\nugets</PackageOutputPath>
```

Can use the technology-agnostic name `dist` (for distributables) instead?

```suggestion
    <PackageOutputPath>.\dist</PackageOutputPath>
```